### ISSUE TITLE
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/lisc/scratch/neurobiology/zimmer/.conda/envs/wbfm/lib/python3.8/site-packages/cv2/qt/plugins" even though it was found.

### ISSUE THREAD

**Issue by Zihaozhai02** [Zimmer-lab/barlow_track#30]
[zhai@login01 barlow_track]$ <wbfm>python "/lisc/scratch/neurobiology/zimmer/zihaozhai/WBFM/barlow_track/barlow_track/scripts/train_barlow_clusterer.py" -p "/lisc/scratch/neurobiology/zimmer/wbfm/TrainedBarlow/barlow_project_AVBHisCl/train_config.yaml"
/lisc/scratch/neurobiology/zimmer/.conda/envs/wbfm/lib/python3.8/site-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: /lisc/scratch/neurobiology/zimmer/.conda/envs/wbfm/lib/python3.8/site-packages/torchvision/image.so: undefined symbol: ZN5torch3jit17parseSchemaOrNameERKNSt7_cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
  warn(f"Failed to load image Python extension: {e}")
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: (1) Create a W&B account
wandb: (2) Use an existing W&B account
wandb: (3) Don't visualize my results
wandb: Enter your choice: 3
wandb: You chose "Don't visualize my results"
ConfigFile  : INFO     =======================================
Project data for directory:
/lisc/scratch/neurobiology/zimmer/zihaozhai/WBFM/project/AVBHisCl/2025-06-06_16-40_AVBhiscl_histamin_2per_worm2-2025-06-06
With raw data in directory:
/lisc/scratch/neurobiology/zimmer/zihaozhai/WBFM/raw/AVBhiscl/2025-06-06_16-40_AVBhiscl_histamin_2per_worm2
See self.worm_posture_class for information on behavioral parameters
Found the following data files:
============Preprocessed Videos========
red_data:                 True
green_data:               True
============Segmentation===============
raw_segmentation:         True
colored_segmentation:     True
============Tracking===================
tracklets:                True
final_tracks:             True
manual_tracking:          True
============Traces=====================
traces:                   True

Preparing cropped volumes...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 200/200 [00:47<00:00,  4.17it/s]
Starting training. Stats in folder: /lisc/scratch/neurobiology/zimmer/wbfm/TrainedBarlow/barlow_project_AVBHisCl
wandb: Tracking run with wandb version 0.18.7
wandb: W&B syncing is set to offline in this directory.
wandb: Run wandb online or set WANDB_MODE=online to enable cloud syncing.
{"epoch": 0, "step": 0, "loss": 8.15493392944336, "time": 26}
QObject::moveToThread: Current thread (0x560191a6b7c0) is not the object's thread (0x560191c44d20).
Cannot move to target thread (0x560191a6b7c0)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/lisc/scratch/neurobiology/zimmer/.conda/envs/wbfm/lib/python3.8/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, minimal, minimalegl, offscreen, vnc, webgl.

Aborted (core dumped)

**Comment by Charles-Fieseler-Vienna** [comment]
Are you using a different pretrained path compared to Itamar? Do you know why you are not getting the error #29 ?

**Comment by Charles-Fieseler-Vienna** [comment]
Can you try to run this before and then try again?

```
export DISPLAY=:0
```

**Comment by Zihaozhai02** [comment]
> Are you using a different pretrained path compared to Itamar? Do you know why you are not getting the error [#29](https://github.com/Zimmer-lab/barlow_track/issues/29) ?

we are using the same which is /lisc/scratch/neurobiology/zimmer/wbfm/TrainedBarlow/barlow_ZIM2165_Gcamp7b_worm1-2022_11_28_from_search/trial_13/resnet50.pth, and no i don't know why

**Comment by Zihaozhai02** [comment]
> Can you try to run this before and then try again?
> 
> ```
> export DISPLAY=:0
> ```

will try asap

**Comment by Charles-Fieseler-Vienna** [comment]
@Zihaozhai02 did this work?

**Comment by Zihaozhai02** [comment]
> @Zihaozhai02 did this work?


It is running and I could see that it is generating epochs. Itamar told me that you figured something to improve in the code.  So I will rerun after it. 
Sorry for the late response.

**Comment by Charles-Fieseler-Vienna** [comment]
This is the solution seemingly:
```
export DISPLAY=:0
```
